### PR TITLE
Add movils and find by city

### PR DESCRIPTION
**Pull request by aryehbeitz**

### CODE REVIEW

**Comment by aryehbeitz** [comment]
[the page to manage מובילי קהילות crashes](https://trello.com/c/yWYyTc2b/203-the-page-to-manage-%D7%9E%D7%95%D7%91%D7%99%D7%9C%D7%99-%D7%A7%D7%94%D7%99%D7%9C%D7%95%D7%AA-crashes)

File: app/controllers/managers_controller.rb [hunk 1]
```diff
@@ -2,18 +2,8 @@ class ManagersController < ApplicationController
   before_filter :set_manager, only: [:show, :edit, :update, :destroy, :remove_city, :filter_hosts, :export_hosts, :export_witnesses, :export_guests]
   before_filter :is_admin, only: [:index, :export_hosts, :export_witnesses, :export_guests]
   before_filter :correct_manager, only: [:show, :export]
-  # before_action :authenticate_user!
```
Comment (opinyo) [inline app/controllers/managers_controller.rb]: Why is it removed?

File: app/controllers/managers_controller.rb [hunk 2]
```diff
@@ -109,12 +107,33 @@ def destroy
     render :json => @manager.to_json
   end
 
+  def add_city
+    @manager = Manager.find(params[:id])
+    CommunityLeadership.where(manager_id: @manager.id, city_id: params[:city_id]).first || CommunityLeadership.create(manager_id: @manager.id, city_id: params[:city_id])
```
Comment (opinyo) [inline app/controllers/managers_controller.rb]: `CommunityLeadership.create` might fail. Consider handling this case.

File: app/controllers/managers_controller.rb (hunk 1 above)
Comment (aryehbeitz) [reply app/controllers/managers_controller.rb]: it's being handled in a different way

File: app/controllers/managers_controller.rb (hunk 2 above)
Comment (aryehbeitz) [reply app/controllers/managers_controller.rb]: done

**Pull request merged**